src/builder/affordance.rs

```diff
@@ -19,7 +19,7 @@ use super::{
     Error, FormBuilder, MultiLanguageBuilder,
 };
 
-pub struct AffordanceBuilder<Affordance> {
+pub(super) struct AffordanceBuilder<Affordance> {
```

Comment:
This does not seem to be strictly necessary -- even if it is probably better to have `pub(super)`

Reply:
It is not user-usable as it is, so it is better to hide it (and have less items to document ^^)

Reply:
Uh, sorry -- this is my mind that tricks me: **yours** was the intended thing, not mine :sweat_smile: